platform/v1alpha1/types.go

```diff
@@ -0,0 +1,76 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	TypeSourced = "Sourced"
+	TypeApplied = "Applied"
+
+	ReasonSourceFailed     = "SourceFailed"
+	ReasonSourceSuccessful = "SourceSuccessful"
+	ReasonApplyFailed      = "ApplyFailed"
+	ReasonApplySuccessful  = "ApplySuccessful"
+)
```

Comment:
Joel had mentioned that we can house these in the downstream o/platform-operators repository vs. in this repository in the EP: https://github.com/openshift/enhancements/pull/1170#discussion_r934404076.

Reply:
Yeah I would pop these constants in your own repo rather than the API unless you expect them to be used a lot across other projects